eng/tools/openapi-diff-runner/src/log.ts

```diff
@@ -130,3 +130,37 @@ export async function logGroup<T>(title: string, content: () => Promise<T> | T):
     logMessage("", LogLevel.EndGroup);
   }
 }
+
+/**
+ * Maximum safe log line length for GitHub Actions (64KB - some buffer)
+ */
+const MAX_LOG_LINE_LENGTH = 60 * 1024; // 60KB to leave some buffer
```

Comment:
what if a line exceeds the limit?

Reply:
Truncate the latter content.

Reply:
Is the default behavior good enough, we don't need to special-case this?